LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
MIT license?

Reply:
Oops. Nice catch. I didn't notice this part.